envoy.bazelrc

```diff
@@ -64,6 +64,7 @@ common --experimental_allow_tags_propagation
 build:linux --copt=-fdebug-types-section
 build:linux --copt=-fPIC
 build:linux --copt=-Wno-deprecated-declarations
+build:linux --copt=-fno-limit-debug-info
```

Comment:
The PR description is currently empty. Can you add to the PR description of 1) why this option needs to be added; 2) pros and cons of adding this option?